### PR TITLE
Split `read_multi_values_bytes` queries for ScyllaDb in chunks of 100

### PR DESCRIPTION
**Pull request by MathieuDutSik**
## Motivation

The recently merged PR about the `read_multi_values_bytes` for ScyllaDb is sometimes the limit that queries need to have a size of at most 100.

## Proposal

The solution is to split queries into chunks of at most 100. This is done with the `join` logic that was used before.

## Test Plan

A test with 150 entries has been added to the CI.

## Release Plan

Not relevant.

## Links

- [reviewer checklist](https://github.com/linera-io/linera-protocol/blob/main/CONTRIBUTING.md#reviewer-checklist)

### CODE REVIEW

File: linera-views/src/scylla_db.rs [hunk 1]
```diff
@@ -420,7 +420,16 @@ impl ReadableKeyValueStore<ScyllaDbContextError> for ScyllaDbStoreInternal {
         }
         let store = self.store.deref();
         let _guard = self.acquire().await;
-        store.read_multi_values_internal(keys).await
+        // This is the maximal size of a query in ScyllaDb with current settings.
+        let max_size_queries = 100;
```
Comment (ma2bd) [inline linera-views/src/scylla_db.rs:424]: Should we create a top-level constant?

Comment (MathieuDutSik) [reply linera-views/src/scylla_db.rs]: Right, corrected.

File: linera-views/src/scylla_db.rs [hunk 2]
```diff
@@ -15,6 +15,9 @@
 //! [trait1]: common::KeyValueStore
 //! [trait2]: common::Context
 
+/// Fundamental constant in ScyllaDB: The maximum size of a multi keys query
```
Comment (afck) [inline linera-views/src/scylla_db.rs:18]: Do you have a link to the documentation? I couldn't find it anywhere. Is it the default value for `--max-clustering-key-restrictions-per-query`?

**Review by afck — APPROVED** [review]

File: linera-views/src/scylla_db.rs (hunk 2 above)
Comment (MathieuDutSik) [reply linera-views/src/scylla_db.rs:18]: I could not find it either in the `scylla.yaml` nor in the CQL documentation. I will merge it and look at the scyllaDB source code.

**Pull request merged**